packages/button/src/Button.js

```diff
@@ -10,70 +10,73 @@ import StyledButton from "./styledComponents/Button";
 
 const EnhancedButton = disabledTooltip(StyledButton);
 
-const Button = ({
-  disabled,
-  icon,
-  rightIcon,
-  text,
-  children,
-  noTooltip,
-  ...buttonProps
-}) => {
-  // @FIXME revisit font icon assets on farmblocks
-  const iconName = buttonProps.loading ? "wg-loading" : icon;
+// eslint-disable-next-line
```

Comment:
Can we disable just the rule that's affecting us?
Also, let's add a comment on why it had to be disabled as it's not obvious from the code.

Reply:
Sure!